version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.0.1-SNAPSHOT"
```

Comment:
Is this file be refered somewhere else

Reply:
separate version.sbt is required for release automation via sbt-release (https://github.com/sbt/sbt-release)